<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPage</class>
 <widget class="QWidget" name="MenuPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="userInfoLabel">
       <property name="text">
        <string>User data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="pager">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="MenuSubpageOne" name="page1"/>
       <widget class="MenuSubpageTwo" name="page2"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuSubpageOne</class>
   <extends>QWidget</extends>
   <header location="global">MenuSubpageOne.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MenuSubpageTwo</class>
   <extends>QWidget</extends>
   <header location="global">MenuSubpageTwo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
